notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb

```diff
@@ -0,0 +1,1648 @@
+      "Find more information about API v2.0 and IR v11 at https://docs.openvino.ai/2023.0/openvino_2_0_transition_guide.html\n",
+      "[ SUCCESS ] Generated IR version 11 model.\n",
+      "[ SUCCESS ] XML file: /home/asuvorov/projects/openvino/openvino_notebooks/notebooks/121-convert-to-openvino/model/distilbert.xml\n",
+      "[ SUCCESS ] BIN file: /home/asuvorov/projects/openvino/openvino_notebooks/notebooks/121-convert-to-openvino/model/distilbert.bin\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Model Optimizer CLI\n",
+    "\n",
+    "! mo --input_model model/distilbert.onnx --output_dir model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
+      "To disable this warning, you can either:\n",
+      "\t- Avoid using `tokenizers` before the fork if possible\n",
+      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Python conversion API\n",
+    "from openvino.tools.mo import convert_model\n",
+    "\n",
+    "ov_model = convert_model(ONNX_NLP_MODEL_PATH)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Model conversion parameters\n",
+    "\n",
+    "Both Python conversion API and Model Optimizer command-line tool provide next capabilities:\n",
+    "* overriding original input shapes for model conversion with `input` and `input_shape` parameters. [Setting Input Shapes guide](https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model.html).\n",
+    "* cutting off unwanted parts of a model (such as unsupported operations and training sub-graphs) using the `input` and `output` parameters to define new inputs and outputs of the converted model. [Cutting Off Parts of a Model guide](https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_prepare_model_convert_model_Cutting_Model.html).\n",
+    "* inserting additional input pre-processing sub-graphs into the converted model by using the `mean_values`, `scales_values`, `layout`, and other parameters. [Embedding Preprocessing Computation article](https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_Additional_Optimization_Use_Cases.html).\n",
+    "* compressing the model weights (for example, weights for convolutions and matrix multiplications) to FP16 data type using `compress_to_fp16` compression parameter. [Compression of a Model to FP16 guide](https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_FP16_Compression.html).\n",
+    "\n",
+    "If the out-of-the-box conversion (only the `input_model` parameter is specified) is not successful, it may be required ti use the parameters mentioned above to override input shapes and cut the model."
```

Comment:
```suggestion
    "If the out-of-the-box conversion (only the `input_model` parameter is specified) is not successful, it may be required to use the parameters mentioned above to override input shapes and cut the model."
```